packages/workbox-background-sync/src/lib/request-queue.js

```diff
@@ -96,6 +96,7 @@ class RequestQueue {
    * @private
    */
   async push({request}) {
+    await this._initializationPromise;
```

Comment:
does it make sense to have a getter for the current queue that does this await?

MY main concern here is that methods exist that use `this._queue` but aren't awaiting on the `initializationPromise` (i.e. `saveQueue()`)